test/common/services/kibana_server/ui_settings.js

```diff
@@ -59,10 +41,18 @@ export class KibanaServerUiSettings {
   }
 
   async replace(doc) {
+    const { payload } = await this._wreck.get('/api/kibana/settings');
```

Comment:
Not related to this PR, but it looks like the comment for the method above (`disableToastAutohide`) says:
> Sets the auto-hide timeout to 1 hour ....

But `360000` values used for settings are missing one `0` I think :)